### PR TITLE
NXP-26002: blob enrichers + appLinks enricher

### PR DESCRIPTION
**Pull request by nelsonsilva**
Depends on https://github.com/nuxeo/nuxeo/pull/2481, https://github.com/nuxeo/nuxeo/pull/2474, https://github.com/nuxeo/nuxeo-elements/pull/37 and https://github.com/nuxeo/nuxeo-ui-elements/pull/291

### CODE REVIEW

File: elements/nuxeo-app.html [hunk 1]
```diff
@@ -1160,36 +1160,50 @@
...
-          'X-NXfetch.task': 'actors',
-          'X-NXtranslate.directoryEntry': 'label'
-        };
+
+        var headers = {
+          'translate-directoryEntry': 'label',
+        }
+
+        var fetch = Nuxeo.UI && Nuxeo.UI.config && Nuxeo.UI.config.fetch;
+        fetch = fetch|| {document: ['properties']};
```
Comment (Gabez0r) [inline elements/nuxeo-app.html]: missing space beffore `||`

File: elements/nuxeo-app.html [hunk 2]
```diff
@@ -1160,36 +1160,50 @@
...
+        // ensure we fetch lock details
+        fetch.document = fetch.document || [];
+        fetch.document.push('lock');
```
Comment (Gabez0r) [inline elements/nuxeo-app.html]: We should double check `lock` isn't already there:
```suggestion
        if (!fetch.document.includes('lock')) {
          fetch.document.push('lock');
        }
```

File: elements/nuxeo-app.html [hunk 3]
```diff
@@ -1160,36 +1160,50 @@
...
+        // add required fetchers
+        Object.assign(fetch, {directoryEntry: ['parent'], task: ['actors']});
```
Comment (Gabez0r) [inline elements/nuxeo-app.html]: Values set for `directoryEntry` and `task` fetchers on `web-ui-properties.xml` will be overridden here. We could check if they exist and append this values if not already present.

**Comment by nelsonsilva** [comment]
https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-nsilva/57/

**Pull request merged**